ros_buildfarm/common.py

```diff
@@ -144,7 +144,7 @@ def get_distribution_repository_keys(urls, key_files):
     for url, key_file in distribution_repositories:
         print('  %s%s' % (url, ' (%s)' % key_file if key_file else ''))
         with open(key_file, 'r') as h:
-            keys.append(h.read())
+            keys.append(h.read().strip())
```

Comment:
`rstrip`?

Reply:
Sure, but I'm not sure why we'd want leading whitespace either. Updated.

Reply:
Is there a case where we have any?

Reply:
We don't. Defensive programming habits, I'd guess.